CHANGELOG.md

```diff
@@ -0,0 +1,454 @@
+## Release Notes
+
+#### Version 3.4 (July 31st, 2019)
+
+-   [Fix security
+    issue](https://jenkins.io/security/advisory/2019-07-31/#SECURITY-713)
+
+#### Version 3.3 (June 14th, 2019)
+
+-   ![(tick)](docs/images/check.svg){ Fixed: maven-plugin random socket leak, leading to
+    threads leak on slave and master
+    ([JENKINS-57119](https://issues.jenkins-ci.org/browse/JENKINS-57119)
+-   ![(tick)](docs/images/check.svg) Fixed: [java.io](http://java.io).NotSerializableException:
+    The calling thread Thread has no associated
+    channel ([JENKINS-57244](https://issues.jenkins-ci.org/browse/JENKINS-57244)
+-   ![(tick)](docs/images/check.svg) Fixed: Upgrade maven embedder from 3.1.0 to at least
+    3.5.4 ([JENKINS-54530](https://issues.jenkins-ci.org/browse/JENKINS-54530)
+
+#### Version 3.2 (November 30th, 2018)
+
+-   ![(tick)](docs/images/check.svg) Fixed: remove anonymous classes
+    ([JENKINS-53481](https://issues.jenkins-ci.org/browse/JENKINS-53481)
+
+#### Version 3.1.2 (March 27th, 2018)
+
+-   ![(tick)](docs/images/check.svg) Fixed: JEP-200 failure to serialize Notifier, used
+    when \<ciManagement\> specifies email configuration. (  [
+    JENKINS-50251](https://issues.jenkins-ci.org/browse/JENKINS-50251) -
+    Getting issue details... STATUS  ) - Take 2 - fix a potential NPE
```

Comment:
WIll need to remove the jira status link text throughout the changelog: `Getting issue details... STATUS`